bundles/AdminBundle/Resources/public/js/pimcore/object/classes/layout/panel.js

```diff
@@ -43,6 +43,14 @@ pimcore.object.classes.layout.panel = Class.create(pimcore.object.classes.layout
             ]
         });
 
+        var labelAligns = Ext.create('Ext.data.Store', {
```

Comment:
why not support all 4?

https://docs.sencha.com/extjs/6.0.1/modern/Ext.field.Field.html#cfg-labelAlign

Reply:
For me not all 4 made sense. But we can add all 4 options, yes :)

Reply:
What about other container types? As far as I see it only panel and localizedfields is covered.

Reply:
I add right/bottom options and cover fieldcontainer/fieldset container types.
right/bottom seems not working. Is there something missing in the theme?

Reply:
In that case I would say leave it as it was before for now :-)
As far as I read the code this is not properly supported.
So just top and left. 

ext-all-debug.js lines ~151698

Reply:
In the extjs demo from 6.0.1 it works. Is there a chance to update to this version?

Reply:
What and where exatcly. We are on 6.0.1.

Reply:
The comment on top of ext-all-debug.js says 6.0.0.640
https://docs.sencha.com/extjs/6.0.1/modern/Ext.field.Text.html
Here you can execute code. Here the labelAlign options right/bottom works:
```
Ext.create('Ext.form.Panel', {
    fullscreen: true,
    items: [
        {
            xtype: 'fieldset',
            title: 'Enter your name',
            items: [
                {
                    xtype: 'textfield',
                    label: 'First Name',
                    name: 'firstName',
                    labelAlign: 'right'
                },
                {
                    xtype: 'textfield',
                    label: 'Last Name',
                    name: 'lastName',
                    labelAlign: 'bottom'
                }
            ]
        }
    ]
});
```

Reply:
You are referring to the modern toolkit. However, we are on "classic".
Please compare https://docs.sencha.com/extjs/6.0.1/classic/Ext.form.field.Text.html
There it doesn't work

Reply:
Thanks for the explanation 👍 
I will remove the options